setup/README.md

```diff
@@ -244,7 +244,7 @@ Your signed images were allowed through, in contrast to those unsigned ones with
 
 ## Cleanup
 
-If you want to remove Connaisseur, run `make uninstall`. We do not delete the namespace, since you might have deployed things in there, which we would inadvertently delete. If you want to get rid of the namespace, run `kubectl delete ns WHATEVEROURDEFAULTIS`.
+If you want to remove Connaisseur, run `make uninstall`.
```

Comment:
Why are you dropping the namespace cleanup hint again?

Reply:
cuz of this:
https://github.com/sse-secure-systems/connaisseur/pull/90#discussion_r576136886